src/main/java/mekanism/client/gui/GuiDigitalMinerConfig.java

```diff
@@ -520,64 +250,27 @@ public void keyTyped(char c, int i) throws IOException {
         }
     }
 
-    private void setRadius() {
+    private void setRadius(boolean playSound) {
         if (!radiusField.getText().isEmpty()) {
             int toUse = Math.max(0, Math.min(Integer.parseInt(radiusField.getText()), MekanismConfig.current().general.digitalMinerMaxRadius.val()));
-            TileNetworkList data = TileNetworkList.withContents(6, toUse);
-            Mekanism.packetHandler.sendToServer(new TileEntityMessage(tileEntity, data));
+            sendDataFromClick(TileNetworkList.withContents(6, toUse), playSound ? SoundEvents.UI_BUTTON_CLICK : null);
             radiusField.setText("");
         }
     }
 
-    private void setMinY() {
+    private void setMinY(boolean playSound) {
         if (!minField.getText().isEmpty()) {
             int toUse = Math.max(0, Math.min(Integer.parseInt(minField.getText()), tileEntity.maxY));
-            TileNetworkList data = TileNetworkList.withContents(7, toUse);
-            Mekanism.packetHandler.sendToServer(new TileEntityMessage(tileEntity, data));
+            sendDataFromClick(TileNetworkList.withContents(7, toUse), playSound ? SoundEvents.UI_BUTTON_CLICK : null);
             minField.setText("");
         }
     }
 
-    private void setMaxY() {
+    private void setMaxY(boolean playSound) {
```

Comment:
The sound didn't get removed as much as some code dedupliacated. The case when playSound is false is when hitting enter. (I believe it was because technically you are not clicking the button, not sure that is how it was before my changes so I left it like that)

Reply:
ah my bad, misread the diff and thought the sound was already handled in that method. Tho tbh it would be a better experience if it did, so that you know something got sent to server

Reply:
Ya, that sounds like a reasonable change.